src/test/java/uk/gov/hmcts/darts/datastore/DataManagementServiceImplTest.java

```diff
@@ -43,9 +48,10 @@ class DataManagementServiceImplTest {
 
     @BeforeEach
     void beforeEach() {
-        when(dataManagementFactory.getBlobContainerClient(Mockito.anyString(), Mockito.notNull())).thenReturn(blobContainerClient);
-        when(dataManagementFactory.getBlobServiceClient(Mockito.notNull())).thenReturn(serviceClient);
-        when(dataManagementConfiguration.getBlobStorageAccountConnectionString()).thenReturn("connection");
+        lenient().when(dataManagementFactory.getBlobContainerClient(Mockito.anyString(), Mockito.notNull())).thenReturn(blobContainerClient);
+        lenient().when(dataManagementFactory.getBlobServiceClient(Mockito.notNull())).thenReturn(serviceClient);
+        lenient().when(dataManagementConfiguration.getBlobStorageAccountConnectionString()).thenReturn("connection");
+        lenient().when(dataManagementFactory.getBlobClient(any(), any())).thenReturn(blobClient);
     }
 
     @Test
```

Comment:
Is it worth verying that the delete blob `verify(blobClient).deleteIfExistsWithResponse` line is not hit from the existing happy path test(s)?